flink-rpc/flink-rpc-akka/src/main/resources/META-INF/NOTICE

```diff
@@ -29,7 +29,7 @@ The following dependencies all share the same BSD license which you find under l
 - org.scala-lang:scala-library:2.12.7
 - org.scala-lang:scala-reflect:2.12.7
 - org.scala-lang.modules:scala-java8-compat_2.12:0.8.0
-- org.scala-lang.modules:scala-parser-combinators_2.12:1.0.4
+- org.scala-lang.modules:scala-parser-combinators_2.12:1.1.2
```

Comment:
This has changed because the now-removed `scalatest` also had a transitive dependency on it, and dependency resolution gave it precedence.